spotpy/unittests/test_database.py

```diff
@@ -53,6 +54,7 @@ def test_csv_multiline_false(self):
                      chains=1, save_sim=False)
 
         csv.save(self.like, self.randompar, self.simulations_multi)
+        csv.save(self.like, self.randompar, self.simulations_multi)
```

Comment:
The extra `csv.save` is so that `csvdata` has more than one entry in the list. (It wouldn't make sense to have a single entry in the CSV file anyway, so this doesn't break anything)